proposals/0345-android-12-issues.md

```diff
@@ -165,19 +268,67 @@ private static SdlDeviceListener getSdlDeviceListener(Context context, Bluetooth
 }
 ~~~
 
-We also need to consider interactions with applications supporting older versions of the `SdlRouterService`. If an application using SDL Android 4.11 or older receives an `ACL_CONNECTED` broadcast in the `SdlBroadcastReceiver`, it may try to start up the `SdlRouterService` of an application that is targeting Android 12. If the started `SdlRouterService` does not have Bluetooth permissions, it will not be able to use the `bluetoothTransport`. To solve this issue we can update the `initCheck` within the `SdlRouterService` to check the app's Bluetooth permissions. If the permissions are not granted, we can fail the `initCheck` and try to deploy the next `SdlRouterService`.
+We also need to consider interactions with applications supporting older versions of the `SdlRouterService`. If an application using SDL Android 4.11 or older receives an `ACL_CONNECTED` broadcast in the `SdlBroadcastReceiver`, it may try to start up the `SdlRouterService` of an application that is targeting Android 12. If the started `SdlRouterService` does not have Bluetooth permission, it will not be able to use the `bluetoothTransport`. To solve this issue we can update the `initCheck` within the `SdlRouterService` to check the app's Bluetooth permission. If the permission is not granted, we can fail the `initCheck` and try to deploy the next `SdlRouterService`.
```

Comment:
```suggestion
We also need to consider interactions with applications supporting older versions of the `SdlRouterService`. If an application using SDL Android 4.11 or older receives an `ACL_CONNECTED` broadcast in the `SdlBroadcastReceiver`, it may try to start up the `SdlRouterService` of an application that is targeting Android 12. If the started `SdlRouterService` does not have the Bluetooth permission, it will not be able to use the `bluetoothTransport`. To solve this issue we can update the `initCheck` within the `SdlRouterService` to check the app's Bluetooth permission. If the permission is not granted, we can fail the `initCheck` and try to deploy the next `SdlRouterService`.
```